### PR TITLE
OAI-22: Change label, move filter to review page

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [OAI-23](https://openimis.atlassian.net/browse/OAI-23)

### CODE REVIEW

File: src/components/ClaimFilterWasCategorised.js [hunk 1]
```diff
@@ -59,16 +60,27 @@ class ClaimFilterWasCategorised extends Component {
...
+        const { filters } = this.props;
+        var json_not_exists = (json_ext === null);
+        var prev_claim_param = prev_json_ext['value']
+        var valueExisted = !!prev_claim_param['claim_ai_quality']
+        var valueUpdated = false
+        var wasCategorized = false
+        if (valueExisted) {
+            wasCategorized = prev_claim_param['claim_ai_quality']['was_categorized']
+        }
+        console.log("Was cat: " + wasCategorized)
```
Comment (dragos-dobre) [inline src/components/ClaimFilterWasCategorised.js]: I suppose this is for debugging. Please remove.

File: src/components/ClaimFilterWasCategorised.js [hunk 2]
```diff
@@ -59,16 +60,27 @@ class ClaimFilterWasCategorised extends Component {
...
+
+        return statusChecked && (json_not_exists || valueUpdated)  
     }
 
     _checkRemoveJson(json_ext) {
         var checkboxState = this.state.categorized === false;
-        var jsonHasAttribute = !!json_ext['value']['wasCategorized']
-
+        var jsonHasAttribute = (!!json_ext && !!json_ext['value']['claim_ai_quality'])
+        console.log(JSON.stringify(json_ext), json_ext['value']['claim_ai_quality'])
```
Comment (dragos-dobre) [inline src/components/ClaimFilterWasCategorised.js]: I suppose this is for debugging. Please remove if not required.

File: src/components/ClaimFilterWasCategorised.js [hunk 3]
```diff
@@ -97,7 +109,7 @@ class ClaimFilterWasCategorised extends Component {
                                 onChange={e => this._onWasCategorizedCheckbox()}
                             />
                         }
-                        label="Categorized"
+                        label="Categorized by AI"
```
Comment (dragos-dobre) [inline src/components/ClaimFilterWasCategorised.js]: The label should be added to the translations.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: src/components/ClaimFilterWasCategorised.js (hunk 1 above)
Comment (dborowiecki) [reply src/components/ClaimFilterWasCategorised.js]: Removed

File: src/components/ClaimFilterWasCategorised.js (hunk 2 above)
Comment (dborowiecki) [reply src/components/ClaimFilterWasCategorised.js]: Removed

File: src/components/ClaimFilterWasCategorised.js (hunk 3 above)
Comment (dborowiecki) [reply src/components/ClaimFilterWasCategorised.js]: Moved to translations

**Comment by dborowiecki** [comment]
@dragos-dobre I added requested changes, can you review?

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**